applications/app/services/dotcomrendering/MediaPicker.scala

```diff
@@ -5,28 +5,24 @@ import model.Cors.RichRequestHeader
 import model.{MediaPage, Video, Audio}
 import play.api.mvc.RequestHeader
 import utils.DotcomponentsLogger
-import navigation.NavLinks.media
-import experiments.ActiveExperiments
-import conf.switches.Switches.DCRVideoPages
+import conf.switches.Switches.{DCRAudioPages, DCRVideoPages}
 
 object MediaPicker extends GuLogging {
 
   /** Add to this function any logic for including/excluding an audio/video article from being rendered with DCR
-    *
-    * Currently defaulting to false until we implement in DCR
     */
-  private def dcrCouldRender(mediaPage: MediaPage): Boolean = {
+  private def dcrShouldRender(mediaPage: MediaPage): Boolean = {
     mediaPage.media match {
```

Comment:
Is a media page always either Audio or Video?

Reply:
i'm guessing from https://github.com/guardian/frontend/pull/26207 it is? i'm not sure though

Reply:
I think so too but I'm also not 100%, so it is probably safer to keep this pattern matching in for now.